frontend/src/Dashboard/SwiperCards.js

```diff
@@ -0,0 +1,388 @@
+                </IconContext.Provider>
+                <MeasureCard
+                  titleText="Besugárzás"
+                  text={weatherData.miscData.irradiation}
+                  unit="W/m2"
+                  lastData="Utolsó mért adat: "
+                  footerText={weatherData.date}
+                ></MeasureCard>
+              </div>
+            </div>
+          </SwiperSlide>
+          <SwiperSlide>
+            <div className="zoom">
+              <div className="card">
+                <IconContext.Provider value={{ color: '#c54b3c' }}>
+                  <HiSun size={100} />
+                </IconContext.Provider>
+                <MeasureCard
+                  titleText="Fényegység"
+                  text={weatherData.miscData.lightUnit}
+                  unit="cd"
+                  lastData="Utolsó mért adat: "
+                  footerText={weatherData.date}
+                ></MeasureCard>
+              </div>
+            </div>
+          </SwiperSlide>
+          <SwiperSlide>
+            <div className="zoom">
+              <div className="card">
+                <IconContext.Provider value={{ color: '#c54b3c' }}>
+                  <RiBattery2ChargeLine size={100} />
+                </IconContext.Provider>
+                <MeasureCard
+                  titleText="Napelem töltő feszültség"
+                  text={weatherData.batteryData.solarCellChargingVoltage}
+                  unit="V"
+                  lastData="Utolsó mért adat: "
+                  footerText={weatherData.date}
+                ></MeasureCard>
+              </div>
+            </div>
+          </SwiperSlide>
+          <SwiperSlide>
+            <div className="zoom">
+              <div className="card">
+                <IconContext.Provider value={{ color: '#c54b3c' }}>
+                  <GiCarBattery size={100} />
+                </IconContext.Provider>
+                <MeasureCard
+                  titleText="Külső akkufeszültség"
+                  text={weatherData.batteryData.externalBatteryVoltage}
+                  unit="V"
+                  lastData="Utolsó mért adat: "
+                  footerText={weatherData.date}
+                ></MeasureCard>
+              </div>
+            </div>
+          </SwiperSlide>
+          <SwiperSlide>
+            <div className="zoom">
+              <div className="card">
+                <IconContext.Provider value={{ color: '#c54b3c' }}>
+                  <GiCarBattery size={100} />
+                </IconContext.Provider>
+                <MeasureCard
+                  titleText="Belső akkufeszültség"
+                  text={weatherData.batteryData.internalBatteryVoltage}
+                  unit="V"
+                  lastData="Utolsó mért adat: "
+                  footerText={weatherData.date}
+                ></MeasureCard>
+              </div>
+            </div>
+          </SwiperSlide>
+        </Swiper>
+      </div>
+    )
+  );
+}
```

Comment:
Ezen lehetne még reszelni, nagyon hosszú és sok a duplikáció...

Reply:
Én a SwiperSlide-ot a gyerekeivel együtt kiszervezném egy komponensbe és props-al etetném be a dinamikus adatokat. Az ikont is át lehet adni props-ban, a többi adat meg egyszerű string.

Reply:
Ezen kívül egy array map-el renderelném, az array meg lehetne valami ilyesmi:
[{
title: 'Belső akkufeszültség',
unit: 'V',
value: weatherData.batteryData.internalBatteryVoltage
icon: GiCarBattery
}, 
...
]